### PR TITLE
Migrated imports of Apollo Client React components to 'apollo-client' package

### LINKED ISSUE #11: Migrate imports of Apollo Client React components
As `apollo-react` package is deprecated, we have to migrate the imports from `apollo-react` to classes in `apollo-client` direclty.

From the deprecation notice on `apollo-react`
> old: @apollo/react-components --> new: @apollo/client/react/components
> old: @apollo/react-hoc --> new: @apollo/client/react/hoc
> old: @apollo/react-ssr --> new: @apollo/client/react/ssr
> old: @apollo/react-testing --> new: @apollo/client/testing
> old: @apollo/react-hooks --> new: @apollo/client

Further migration information on **Apollo** website.
https://www.apollographql.com/docs/react/migrating/apollo-client-3-migration/#updating-imports

This will also remove the package unmet warnings from `apollo-react`.

```
npm WARN @apollo/react-common@3.1.4 requires a peer of graphql@^14.3.1 but none is installed. You must install peer dependencies yourself.
npm WARN @apollo/react-components@3.1.5 requires a peer of graphql@^14.3.1 but none is installed. You must install peer dependencies yourself.
npm WARN @apollo/react-hoc@3.1.5 requires a peer of graphql@^14.3.1 but none is installed. You must install peer dependencies yourself.
npm WARN @apollo/react-hooks@3.1.5 requires a peer of graphql@^14.3.1 but none is installed. You must install peer dependencies yourself.
npm WARN graphql-tag@2.10.3 requires a peer of graphql@^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN react-apollo@3.1.5 requires a peer of graphql@^14.3.1 but none is installed. You must install peer dependencies yourself.
```

### PR DESCRIPTION
**Pull request by MonikaCat**
Fixes #11

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -6,26 +6,23 @@ import Layout from '../components/Layout';
...
+import { InMemoryCache } from '@apollo/client/cache';
+import possibleTypes from '../possibleTypes.json'
 
-const fragmentMatcher = new IntrospectionFragmentMatcher({
-  introspectionQueryResultData
-});
-
-const fetch = require('node-fetch').default;
-const cache = new InMemoryCache({ fragmentMatcher });
+const fetch = require('cross-fetch');
```
Comment (kwunyeung) [inline pages/_app.tsx:15]: Better write as importing ES module

https://github.com/node-fetch/node-fetch#loading-and-configuring-the-module

File: package.json [hunk 2]
```diff
@@ -7,20 +7,20 @@
     "dev-next": "next dev",
     "build": "next build",
     "start": "next start",
-    "build-fragment": "node ./schemaQuery.js"
+    "build-fragment": "node ./schemaQuery.js ./possibleTypesQuery.js"
```
Comment (kwunyeung) [inline package.json:10]: How can you run two nodejs files from a single command?

**Comment by MonikaCat** [comment]
There is a difference in the data returned by ./schemaQuery.js  and ./possibleTypesQuery.js , hence we need to create both of those files when the application starts

File: components/transaction/LatestTransactions.tsx [hunk 3]
```diff
@@ -19,7 +19,8 @@ import Paper from "@material-ui/core/Paper";
 import TablePagination from "@material-ui/core/TablePagination";
 import Chips from "../Chips";
 import Divider from "@material-ui/core/Divider";
-import { useQuery } from "@apollo/react-hooks";
+// import { gql } from "apollo-boost";
```
Comment (kwunyeung) [inline components/transaction/LatestTransactions.tsx:22]:
```suggestion
```

**Pull request merged**